frontend/src/libs/components/article-card/article-card.tsx

```diff
@@ -0,0 +1,75 @@
+import ArticlePreview from '~/assets/img/article-preview.png';
+import { AppRoute } from '~/libs/enums/app-route.enum.js';
+
+import { Avatar } from '../avatar/avatar.js';
+import { Icon, Link } from '../components.js';
+import styles from './styles.module.scss';
+
+const ArticleCard: React.FC = () => (
+  <article className={styles.article}>
+    <div className={styles.header}>
+      <div className={styles.info}>
+        <Link to={AppRoute.PROFILE}>
+          <Avatar username="Nolan Sarls" avatarUrl={null} />
+        </Link>
+        <span className={styles.publisherName}>Nolan Sarls</span>
+        <span className={styles.publicationTime}>May 28</span>
+        <span className={styles.publicationTime}>7 min read</span>
+      </div>
+      <Icon iconName="favorite" className={styles.favoriteIcon} />
+    </div>
+    <div className={styles.body}>
+      <div>
+        <h4 className={styles.title}>
+          Modern Full-Stack Developer Tech Stack 2021
+        </h4>
+        <p className={styles.content}>
+          The developer technology landscape changes all the time as new tools
+          and technologies are introduced. After having lots of interviews and
+          reading through countless job descriptions on job boards I think this
+          is a great modern tech stack for JavaScript developers in 2021.
```

Comment:
I believe, that these data mocks should be removed from the component when you send it for review and replaced with props (card content, reactions number, etc.)